readme/Readme.scalatex

```diff
@@ -782,6 +1000,10 @@
             "com.lihaoyi" %%% "scalaparse" % "@fastparse.Constants.version"
 
 @sect{Change Log}
+    @sect{0.4}
+        @ul
+            @li
+                A lot of things :)
```

Comment:
Let's take the effort to flesh this out =) This will basically be the summary of your GSOC work, so you should let everyone know what you did!